src/roadmap/common.py

```
@@ -72,15 +71,14 @@ async def query_rbac(
-    def _fetch_rbac():
-        req = urllib.request.Request(url, headers=headers)
-        with urllib.request.urlopen(req, timeout=30) as response:
-            return json.load(response)
+    req = urllib.request.Request(
+        f"{settings.rbac_url}/api/rbac/v1/access/?{urllib.parse.urlencode(params, doseq=True)}",
+        headers=headers,
+    )
 
     try:
-        data = await asyncio.to_thread(_fetch_rbac)
+        with urllib.request.urlopen(req) as response:
```

Ruff (0.16.1)

[warning] Async functions should not call blocking HTTP methods

(ASYNC210)